packages/libc6.rb

```diff
@@ -0,0 +1,67 @@
+require 'package'
+
+class Libc6 < Package
+  description 'Debian libc6 Library version 2.24'
```

Comment:
Why do we need Debian's version of the library?

Reply:
Debian is "The universal operating system" and I couldn't find crti.o anywhere else.

Reply:
Debian is just a shell on top of the Linux kernel.  Chrome OS is the operating system, not Debian.  We really should be using the canonical source code without any Debianisms.

Reply:
The source code IS glibc. You need glibc to compile glibc.

Reply:
I would rather use the existing 2.23 version to compile 2.27 instead of this if we can.

Reply:
We'd have to recompile 2.23 for canary too, then. Case statement?

Reply:
How about we use 2.27 to build 2.27 after we get this done? 2.27 will override Debian's files anyway.